.eslintrc.js

```diff
@@ -1,15 +1,12 @@
 module.exports = {
   root: true,
   parserOptions: {
-    ecmaVersion: 2020
+    ecmaVersion: 'latest'
   },
   extends: [
     'plugin:vue/vue3-recommended',
     '@vue/airbnb'
   ],
-  env: {
-    node: true
-  },
```

Comment:
`env` tells ESLint which variables are global variables so it doesn't flag them. However, Frontend isn't a Node project, so specifying `node` here doesn't make sense. The actual global variables that we use are listed below under `globals`.